Two errors running qemu

See the two errors

    00000000000005F2 - 005F5018 - [ERROR] Threads: WorkerThread: Exception: Data abort at 0020A228

    00000000000006AC - 00638CE8 - [ERROR] MMC: SD Switch Highspeed host does not support Highspeed

in https://github.com/markfirmware/ultimc/blob/qemu-data-error/run-data-error.log

--- a/armall.pas
+++ b/armall.pas
@@ -19,7 +19,7 @@ uses
   , SysUtils
   , Classes
   , Ultibo
-  , DWCOTG, Keyboard
+  , Keyboard
   , FileSystem, MMC, FATFS
   , RemoteShell, ShellForth
 

--- a/fipq.lpr
+++ b/fipq.lpr
@@ -7,9 +7,9 @@ program fipq;
 
 uses
   {$ifdef BUILD_MODE_QEMUVPB} QEMUVersatilePB,  {$endif}
-  {$ifdef BUILD_MODE_RPI    } BCM2835, BCM2708, {$endif}
-  {$ifdef BUILD_MODE_RPI2   } BCM2836, BCM2709, {$endif}
-  {$ifdef BUILD_MODE_RPI3   } BCM2837, BCM2710, {$endif}
+  {$ifdef BUILD_MODE_RPI    } RaspberryPi,      {$endif}
+  {$ifdef BUILD_MODE_RPI2   } RaspberryPi2,     {$endif}
+  {$ifdef BUILD_MODE_RPI3   } RaspberryPi3,     {$endif}
   //QEMUVersatilePB
   armall
 

